client/src/app/features/admin-features/admin-panel/components/manage-user/manage-user.component.ts

```
@@ -3,58 +3,58 @@ import { AdminService } from '../../admin.service';
-  }
+      this.bsModalRef = this.modalService.show(
+        RolesModalComponent,
+        initialState
+      );
 
-  getUsersWithRoles() {
-    this.adminService.getUserWithRoles().subscribe({
-      next: (users) => (this.users = users),
+      this.bsModalRef.onHide?.subscribe({
+        next: () => {
+          if (this.bsModalRef.content && this.bsModalRef.content.rolesUpdated) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)